classes/02class/exercises/c02-network05/frdvo/README.md

```diff
@@ -0,0 +1,42 @@
+# c02-network05
+
+## Commands Execution Output
+
+- Commands for creating the internet gateway:
+```
+#Create Internet Gateway
+aws ec2 create-internet-gateway
+
+#Tag Internet Gateway
+aws ec2 create-tags --resources igw-0116304685cc6f00b --tags Key=Name,Value=devopsacademy-igw
+
+#Attach Internet Gateway
+aws ec2 attach-internet-gateway --internet-gateway-id igw-0116304685cc6f00b --vpc-id vpc-0a2b7db4956438d22
+```
+Add your commands and their outputs here
+```bash
+ 🐳 :: 📦 2  📑 :: 1   👉 /home/fer/repos/academy/classes/02class/exercises/c02-network05  
```

Comment:
that's a lot of info and emojis 😆 
can you just add the formatted output below the respective command just above?

Reply:
All good mate.
I'm excited using Cascadia Code PL, but I can't see some characters here. I will send clear the next PR.